src/CSF.SRDashboard.Client/PageValidators/SearchErrorObj.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace CSF.SRDashboard.Client.PageValidators
+{
+    public enum errorType
+    {
+        CRITERIA,
+        NO_RESULT
+    }
+    public class SearchErrorObj
```

Comment:
No need to shorten it it's better to use the full word than abbreviations just for increased clarity.
```suggestion
    public class SearchErrorObject
```